domain-xample/xample-workflows/src/main/java/gov/va/vro/routes/xample/BieXampleRoutes.java

```diff
@@ -49,7 +50,15 @@ void configureRouteToSaveContentionEventToDbFromQueue(final String queue) {
               body.setStatus(200);
               exchange.getMessage().setBody(body);
             })
-        .log("Saved Contention Event to DB  ${exchange.pattern}: body ${body.getClass()}: ${body}");
+        .log("Saved Contention Event to DB  ${exchange.pattern}: body ${body.getClass()}")
+        .process(
+            exchange -> {
+              final BieMessagePayload body = exchange.getMessage().getBody(BieMessagePayload.class);
+              final ObjectMapper objectMapper = new ObjectMapper();
+              final String jsonBody = objectMapper.writeValueAsString(body);
+              exchange.setProperty("jsonBody", jsonBody);
+            })
+        .log("ReceivedMessageEventBody: ${exchangeProperty.jsonBody}");
```

Comment:
If the only purpose for `exchange.setProperty` is to do the logging, just do the logging within the block:
```suggestion
              log.info("ReceivedMessageEventBody: " + jsonBody);
            })
```